TSPL.docc/ReferenceManual/Attributes.md

```diff
@@ -97,7 +97,7 @@ indicates the macro's role:
   on an extension, a type alias, or a type that's nested inside a function,
   or use an extension macro to add an extension that has a peer macro.
 
-The peer, member, and accessor macro roles require a `names:` argument,
+The peer and member macro roles require a `names:` argument,
```

Comment:
I think we should add after this sentence something to the effect of

> The accessor macro role requires a `names:` argument if the
> macro generates a `willSet` or `didSet` property observer. An
> accessor macro that generates property observers can't add
> other accessors, because observers only apply to stored properties.

Reply:
Thank you!